packages/cli/lib/prompts/personalAccessKeyPrompt.js

```diff
@@ -15,10 +15,13 @@ const i18nKey = 'cli.lib.prompts.personalAccessKeyPrompt';
  * Displays notification to user that we are about to open the browser,
  * then opens their browser to the personal-access-key shortlink
  */
-const personalAccessKeyPrompt = async ({ env } = {}) => {
+const personalAccessKeyPrompt = async ({ env, accountId } = {}) => {
```

Comment:
I'm sure this is already on your radar, but just a reminder to update things accordingly after [this](https://github.com/HubSpot/hubspot-cli/pull/691) is merged.

Reply:
Good catch - updated and resolved! Waiting on Erin if she has any updates before merging, thread here: https://hubspot.slack.com/archives/C01GV708YH3/p1652106906292789?thread_ts=1651863988.648859&cid=C01GV708YH3